client/cli/src/commands/runcmd.rs

```diff
@@ -265,7 +265,13 @@ pub struct RunCmd {
 		parse(from_os_str),
 		conflicts_with_all = &[ "password-interactive", "password" ]
 	)]
-	pub password_filename: Option<PathBuf>
+	pub password_filename: Option<PathBuf>,
+
+	/// The size of the instances cache for each runtime.
+	///
+	/// The default value is 2 and the values higher than 256 are ignored.
+	#[structopt(long = "max-runtime-instances", default_value = "2")]
```

Comment:
Any particular reason for setting the default to 2 instead of 8 as it was before?

Reply:
I was looking at the `MAX_RUNTIMES` constant when setting the default value. :sweat_smile: